From a3c8a4c1960dc69b3d59ec30a44f0bea9d6b69c4 Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Date: Mon, 1 Jul 2013 02:32:55 +0200
Subject: [PATCH 29/65] acerhk: remove __dev* attributes

CONFIG_HOTPLUG is going away as an option.  As a result, the __dev*
markings need to be removed.

This change removes the use of __devinit, __devexit_p, __devinitdata,
__devinitconst, and __devexit from these drivers.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
---
 drivers/platform/x86/acerhk.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- a/drivers/platform/x86/acerhk.c
+++ b/drivers/platform/x86/acerhk.c
@@ -1664,7 +1664,7 @@ static struct miscdevice acerhk_misc_dev
 /* }}} */
 
 #ifndef __x86_64__
-static void __devinit model_init(void)
+static void model_init(void)
 {
 	/* set callroutine, features and keymap for model */
 	setup_model_features(acerhk_series);
@@ -1691,9 +1691,9 @@ static void __devinit model_init(void)
 }
 #endif
 
-static int __devexit acerhk_remove(struct platform_device *dev);
+static int acerhk_remove(struct platform_device *dev);
 
-static int __devinit acerhk_probe(struct platform_device *dev)
+static int acerhk_probe(struct platform_device *dev)
 {
 	int ret;
 
@@ -1767,7 +1767,7 @@ static int __devinit acerhk_probe(struct
 	return ret;
 }
 
-static int __devexit acerhk_remove(struct platform_device *dev)
+static int acerhk_remove(struct platform_device *dev)
 {
 	acerhk_proc_cleanup();
 	stop_blinking();
@@ -1794,7 +1794,7 @@ static struct platform_driver acerhk_dri
 		.owner = THIS_MODULE,
 	},
 	.probe = acerhk_probe,
-	.remove = __devexit_p(acerhk_remove),
+	.remove = acerhk_remove,
 #ifdef CONFIG_PM
 	.resume = acerhk_resume,
 #endif
